docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
Do not mount whole application as a volume in the final code in the future, it should be done during development only